source/documentation/cli/migrator.html.md.erb

```diff
@@ -349,3 +346,87 @@ $ cat _index.scss
 @forward "typography";
 @forward "components";
 ```
+
+### Namespace
+
+This migration allows you to easily change the [namespaces][] of the `@use`
+rules in a stylesheet. This is useful if the namespaces that the module migrator
+generates to resolve conflicts are non-ideal, or if you don't want to use the
+default namespace that Sass determines based on the rule's URL.
+
+[namespaces]: ../at-rules/use#choosing-a-namespace
+
+#### `--rename`
+
+You can tell the migrator which namespace(s) you want it to change by passing
+expressions to the `--rename` option.
+
+These expressions are of the form `<old-namespace> to <new-namespace>` or
+`url <rule-url> to <new-namespace>`. In these expressions, `<old-namespace>` and
+`<rule-url>` are [regular expressions][] which match against the entirety of the
+existing namespace or the `@use` rule's URL, respectively.
+
+[regular expressions]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+
+For simple use cases, this just looks like `--rename 'old to new'`, which would
+rename a `@use` rule with the namespace `old` to instead be `new`.
+
+However, you can also do this to accomplish more complicated renames. For
+instance, say that you previously had a stylesheet that looked like this:
+
+```scss
+@import "components/button/lib/mixins";
+@import "components/input/lib/mixins";
+@import "components/table/lib/mixins";
+...
+```
+
+Since all of these URLs would have the default namespace `mixins` when migrated
+to `@use` rules, the module migrator may end up generating something like this:
+
+```scss
+@use "components/button/lib/mixins" as button-lib-mixins;
+@use "components/input/lib/mixins" as input-lib-mixins;
+@use "components/table/lib/mixins" as table-lib-mixins;
+...
+```
+
+This is valid code since the namespaces don't conflict, but they're way more
+complicated than they need to be. The relevant part of the URL is the component
+name, so we can use the namespace migrator to extract that part out.
+
+If we run the namespace migrator with
+`--rename 'url components/(\w+)/lib/mixins to \1'`, we'll end up with:
+
+```scss
+@use "components/button/lib/mixins" as button;
+@use "components/input/lib/mixins" as input;
+@use "components/table/lib/mixins" as table;
+...
+```
+
+The rename script here says to find all of the `@use` rules whose URLs look like
+`components/(\w+)/lib/mixins` (`\w+` in a regular expression means to match any
+word of one or more characters). The `\1` in the output clause means to
+substitute in the contents of the first set of parentheses in the regular
+expression (called a [group][]).
+
+[group]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Ranges
+
+If you wish to try multiple renames, you can pass the `--rename` option multiple
```

Comment:
"Try" is weird here because it implies that the user might fail... maybe "apply"?